framework/include/constraints/AutomaticMortarGeneration.h

```diff
@@ -152,7 +152,64 @@ class AutomaticMortarGeneration : public ConsoleStreamInterface
    */
   bool onDisplaced() const { return _on_displaced; }
 
-public:
+  /**
+   * @return The nodal normals associated with the provided \p secondary_elem
+   */
+  std::vector<Point> getNodalNormals(const Elem & secondary_elem) const;
+
+  /**
+   * Compute the normals at given reference points on a secondary element
+   * @param secondary_elem The secondary element used to query for associated nodal normals
+   * @param xi1_pts The reference points on the secondary element to evaluate the normals at. The
+   * points should only be non-zero in the zeroth entry  because right now our mortar mesh elements
+   * are always 1D
+   * @return The normals
+   */
+  std::vector<Point> getNormals(const Elem & secondary_elem,
+                                const std::vector<Point> & xi1_pts) const;
+
+  /**
+   * Compute the normals at given reference points on a secondary element
+   * @param secondary_elem The secondary element used to query for associated nodal normals
+   * @param 1d_xi1_pts The reference points on the secondary element to evaluate the normals at. The
+   * "points" are single reals corresponding to xi because right now our mortar mesh elements are
+   * always 1D
+   * @return The normals
+   */
+  std::vector<Point> getNormals(const Elem & secondary_elem,
+                                const std::vector<Real> & oned_xi1_pts) const;
+
+  /**
+   * @return The mortar interface coupling
+   */
+  const std::unordered_multimap<dof_id_type, dof_id_type> & mortarInterfaceCoupling() const
+  {
+    return mortar_interface_coupling;
```

Comment:
Member variables without a leading underscore. Not introduced by this PR, but maybe you'd like to jump at the opportunity to add another commit to this? :-D